backends/tofino/cmake/spdlog.cmake

```diff
@@ -1,21 +1,33 @@
 message(STATUS "Fetching spdlog")
 
-include(FetchContent)
```

Comment:
I'm not sure, but shouldn't the include be present in all files that use `FetchContent`, even if it was loaded elsewhere?

Reply:
Possibly, if this file is meant to be run alone. If that is the case we could also promote it to the top of the compiler instead of stashing it away in the Tofino back end.